unit_tests/README.md

```diff
@@ -0,0 +1,16 @@
+# Unit Testing Netguard
+
+## Requirements
+An Android x86/64 emulator with no Google Play
+
+## Running Tests
+* Build the app or just the `vpn-network-imp` module
+* Execute the following:
+```
+cd android-app/vpn-network/vpn-network-impl/build/intermediates/cmake/debug/obj
```

Comment:
This absolute path only exist in your PC, Maybe is best to change to something like
```suggestion
cd ..../vpn-network-impl/build/intermediates/cmake/debug/obj
```